src/main/kotlin/eu/europa/ec/eudi/sdjwt/vc/SdJwtVcVerifier.kt

```diff
@@ -67,18 +67,18 @@ fun interface LookupPublicKeysFromDIDDocument {
 interface SdJwtVcVerifier<out JWT> {
 
     /**
-     * Verifies an SD-JWT (in non enveloped, simple format)
-     * Typically, this is useful to Holder that wants to verify an issued SD-JWT
+     * Verifies an SD-JWT serialized using compact serialization.
```

Comment:
Typically, this is useful to Holder that wants to verify an issued SD-JWT or to Verifier
that wants to verify a presented SD-JWT in case the KB-JWT [must not be present]KeyBindingVerifier.MustNotBePresent]